confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSourceConfig.java

```diff
@@ -161,13 +161,15 @@ public class ConfluentStreamingSourceConfig extends ReferencePluginConfig implem
   private final String kafkaProperties;
 
   @Name(NAME_CLUSTER_API_KEY)
-  @Description("The Confluent API Key.")
+  @Description("The Confluent API Key. Required if connecting with Confluent Cloud")
```

Comment:
nit: Add a `.` at the end for both descriptions.

Reply:
Added `.` at the end for both descriptions. Also updated the `.md` file. e6a2fc04c73eb0bf81df50477d75daa995e44f6f